### ISSUE TITLE
"Error adding card to checkout retrying..." in locales that are not USA

### ISSUE THREAD

**Issue by r27dev** [ianmarmour/nvidia-clerk#157]
When i try to test Nvidia Clerk with 2060 and a locale different from USA i get this message: "Error adding card to checkout retrying..." and the notification is not sent.

**Comment by BROKENMIND617** [comment]
Running on windows command prompt :Notifications working here,(twilio,& telegram) however links to cart from notifications, or add to cart at PC running clerk( or should we all it "scraper now" is not.

"Dang it Nvidia, I just want a card for myself"

update: sorry I literally made a few wrapper scripts Monday  night when I found this clerk. unfortunately, by the time i finished setting everything up the API was down and has just come up so i didn't get to test.

Just so happens  that I  left out -remote command and is the reason i didn't receive a link.
However, now that  -remote is active I'm receiving a 1601****** number that my phone thinks is a phone number.
is this just because add to cart has been disabled?

Confirmed API is back up and can add a 2060 to cart while visiting Nvidia's store directly..
